packages/frontend/vite.config.ts

```diff
@@ -1,33 +1,21 @@
-import { nodeTypes } from "@mdx-js/mdx";
+import mdx from "@mdx-js/rollup";
 import rehypeKatex from "rehype-katex";
-import rehypeRaw from "rehype-raw";
 import remarkMath from "remark-math";
 import { defineConfig } from "vite";
 import solid from "vite-plugin-solid";
 import wasm from "vite-plugin-wasm";
 
-// @ts-expect-error Types are missing.
-// *Also*, this plugin causes Vite 5 to complain about CJS.
-// https://github.com/nksaraf/vinxi/issues/289
-import pkg from "@vinxi/plugin-mdx";
-const { default: mdx } = pkg;
-
 export default defineConfig({
     plugins: [
         wasm(),
-        mdx.withImports({})({
-            jsx: true,
-            jsxImportSource: "solid-js",
-            providerImportSource: "solid-mdx",
-            rehypePlugins: [
-                [rehypeRaw, { passThrough: nodeTypes }],
```

Comment:
@tim-at-topos Was there any reason for including `rehypeRaw`? I didn't look to me like it did anything.